web/server.js

```diff
@@ -39,31 +38,42 @@ module.exports = function(options) {
 
   var server = new Hapi.Server(serverConfig);
 
-  server.connection({
-    host: options.host,
-    port: options.port
-  });
-
   if ( options.logging ) {
-    server.register({
-      register: require('hapi-bunyan'),
+    await server.register({
+      plugin: require('good'),
       options: {
-        logger: require('bunyan').createLogger({
-          name: 'id-webmaker-org',
-          level: options.logLevel
-        })
+        reporters: {
+          bunyan: [{
+            module: 'good-bunyan',
+            args: [{
+              response: '*',
+              log: '*',
+              error: '*',
+              request: '*'
+            }, {
+              logger: require('bunyan').createLogger({
+                name: 'id-webmaker-org',
+                level: 'error'
+              }),
+              levels: {
+                response: 'info',
```

Comment:
feel free to suggest other mappings.
The mapping follows the format:
<good event>: <bunyan level level>

Note the the `log` good event is error because according to their docs, they also use this event for system level errors.